test/test_bson.py

```diff
@@ -802,6 +802,13 @@ def test_vector(self):
         assert float_binary == Binary.from_vector(
             BinaryVector(list_vector, BinaryVectorDtype.FLOAT32)
         )
+        # Confirm kwargs cannot be passed when BinaryVector is provided
+        self.assertRaises(
```

Comment:
Can you write this as a `with self.assertRaises():` so we can confirm a "type-ignore" is detected?

Reply:
Sure can. How do we confirm that a type-ignore is detected?

```
        with self.assertRaises(ValueError):
            Binary.from_vector(
                BinaryVector(list_vector, BinaryVectorDtype.PACKED_BIT, padding),
                dtype=BinaryVectorDtype.PACKED_BIT
            )
```

Reply:
Oh.... I see. This will fail linting without it.

Reply:
Yep that's what I was thinking!

Reply:
TIL. Thanks @ShaneHarvey !  This was a very cool lesson.